pyproject.toml

```diff
@@ -24,8 +24,8 @@ importlib-metadata = {version = "^4.0", python = "<3.8"}
 
 pre-commit = "^2.15.0"
 pygithub = "1.55"
+click = "^8.0.4"
 
-[tool.poetry.dev-dependencies]
```

Comment:
Moved these to normal dependencies, because otherwise the install via extras doesn't work (AFAICT that's a known poetry limitation).

I also committed the `poetry.lock` file here, let me know if I should remove that.

Reply:
Good to know.
I think it is fine to let the poetry,lock here.